src/bindgen/processor/virtual_override.cr

```diff
@@ -397,12 +398,12 @@ module Bindgen
 
       # Name of the jumptable structure for both C++ and Crystal.
       private def jumptable_name(klass)
-        "BgJumptable_#{klass.origin.binding_name}"
```

Comment:
Why replace the method call with a string substitution?

Reply:
This was probably a leftover, `#binding_name` also works (the main difference is whether `::` becomes one or two underscores).